llamacloud_sql_router_ddds/README.md

```
@@ -0,0 +1,43 @@
+
+We use:
+* LlamaIndex for orchestrating the RAG app.
+* SQL to create a database and store city populations.
+* Streamlit to build the UI.
+
+A demo is shown here:
+[Demo](https://github.com/hadenpell/ai-engineering-hub/blob/dailydoseds/llamacloud_sql_router_ddds/demo.mp4) 
+
+## Installation and setup
+
+#### Setup OpenAI
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)